source/crud/tests/unified/aggregate-merge.yml

```diff
@@ -0,0 +1,188 @@
+# This file was created automatically using mongodb-spec-converter.
+# Please review the generated file, then remove this notice.
```

Comment:
Since I didn't really work on the files (e.g. fix formatting, etc.) I decided to leave the note in so the first person that manually changes this file can do their thing.